### DISCUSSION TITLE
Acceso a github copilot

### DISCUSSION THREAD

**Discussion by alvi014** [community/community#145811] [GitHub Education]
### Hi everyone,

Hola amigos.
Quiero obtener los beneficios de github copilot pero no me aceptan la solicitud, he pasado fotos del carnet y la boleta de matrícula pero aún no me dejan... Alguno sabe algo para ver si puedo obtener estos beneficios sin pagar. Gracias

**Comment by samus-aran** [comment]
Hi @alvi014 

You need to ensure you have 2FA enabled and update your billing information. 👍
